reana_server/rest/launch.py

```diff
@@ -218,10 +218,18 @@ def launch(user, url, name="", parameters="{}", specification=None):
         # Start the workflow
         parameters = {"input_parameters": input_parameters}
         publish_workflow_submission(workflow, user.id_, parameters)
+        if validation_warnings:
+            message = (
+                "The workflow has been successfully submitted, but some "
+                "warnings were found during validation. "
+                f"{' '.join([x.message for x in validation_warnings])}"
+            )
+        else:
+            message = "The workflow has been successfully submitted."
         response_data = {
             "workflow_id": workflow.id_,
             "workflow_name": workflow.name,
-            "message": "The workflow has been successfully submitted.",
+            "message": message,
```

Comment:
Maybe we can serialise the warnings in a new `validation_warnings` (name can be changed) field of the reply, so that we can keep it structured? Otherwise it is impossible client/frontend-side to know whether there were warnings when validating the specification without parsing the message.

Something like
```json
{
  "...": "...",
  "validation_warnings": {
    "additional_properties": ["x", "y", "z"]
  }
}
```

We can discuss this in IRL, too.